addons/pos_sale/static/src/app/order_management_screen/sale_order_management_screen/sale_order_management_screen.js

```diff
@@ -288,7 +288,7 @@ export class SaleOrderManagementScreen extends ControlButtonsMixin(Component) {
 
                     let popupTitle = "";
                     let popupInputSuffix = "";
-                    const popupTotalDue = sale_order.amount_total;
+                    const popupTotalDue = sale_order.amount_unpaid;
```

Comment:
It is a very good idea but can you check if we still need the downpayment computation made on lines 278 to 287 ?

Reply:
Also the due balance which is now `this.env.utils.formatCurrency(sale_order.amount_total)` on line 318 should be modified to show the real due amount which would be something like `this.env.utils.formatCurrency(popupTotalDue)`

Reply:
> It is a very good idea but can you check if we still need the downpayment computation made on lines 278 to 287 ?
- @adgu-odoo Yes we can remove it.Thanks for the suggestions.